SYCL/Sampler/basic-rw.cpp

```diff
@@ -0,0 +1,131 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %RUN_ON_HOST %t.out %HOST_CHECK_PLACEHOLDER
```

Comment:
```suggestion
// RUN: %HOST_RUN_PLACEHOLDER %t.out %HOST_CHECK_PLACEHOLDER
```

Reply:
thanks. Is `HOST_RUN_PLACEHOLDER` for the Lit tests that are still on `intel/llvm` as well?

Reply:
We have diffrent placeholders for running on HOST in intel/llvm and here.